crates/valence_command/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "valence_command"
+description = "Command management for Valence"
+readme = "README.md"
+version.workspace = true
+edition.workspace = true
+repository.workspace = true
+documentation.workspace = true
+license.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+bevy_app.workspace = true
+bevy_ecs.workspace = true
+byteorder.workspace = true
+ordered-float.workspace = true
+petgraph.workspace = true
+thiserror.workspace = true
+tracing.workspace = true
+
+valence_server = { workspace = true, optional = true }
+valence_text = { workspace = true, optional = true }
+
+[features]
+valence = ["dep:valence_server", "dep:valence_text"]
```

Comment:
I'm not so sure we should have a `valence` feature flag. The lib makes some deep assumptions about its own use as a bevy plugin for a server, so realistically I don't see it being used elsewhere. The use case is just missing for me.

Feature flags are also annoying to test and easy to screw up, so I like avoiding them if we don't need them.